docs/user/dashboard/tsvb.asciidoc

```diff
@@ -17,15 +18,19 @@ Open *TSVB*, then make sure the required settings are configured.
 
 . On the *New visualization* window, click *TSVB*.
 
-. In *TSVB*, click *Panel options*, then make sure the following settings are configured:
+. In *TSVB*, click *Panel options*, then specify the required *Data* settings.
 
-* *Index pattern*
-* *Time field*
-* *Interval*
+.. From the *Index pattern* dropdown, select the index pattern you want to visualize. 
++
+To visualize an {es} index, open the *Index pattern select mode* menu, deselect *Use only {kib} index patterns*, then select the {es} index from the dropdown.
```

Comment:
You don't select an index, you type it. In that mode we don't have a drop down but an input text.